core/src/main/kotlin/org/evomaster/core/database/ReplaceValuesDeParser.kt

```diff
@@ -0,0 +1,46 @@
+package org.evomaster.core.database
+
+import net.sf.jsqlparser.expression.*
+import net.sf.jsqlparser.util.deparser.ExpressionDeParser
+
+
+class ReplaceValuesDeParser() :
```

Comment:
add some comment about what this class is for

Reply:
ok